src/org/traccar/database/DataManager.java

```diff
@@ -297,7 +297,7 @@ private void initDatabaseSchema() throws SQLException, LiquibaseException {
 
     public User login(String email, String password) throws SQLException {
         User user = QueryBuilder.create(dataSource, getQuery("database.loginUser"))
-                .setString("email", email)
+                .setString("email", email.trim().toLowerCase())
```

Comment:
Query cache optimisation. If two queries came in for the same row it would always look the same to the DB and therefor cached.